spec/features/case_log_spec.rb

```diff
@@ -58,7 +59,7 @@
           when "text"
             fill_in(question.to_s, with: answer)
           when "radio"
-            choose("#{question.to_s.tr('_', '-')}-#{answer}-field")
+            choose("#{question.to_s.tr('_', '-')}-#{answer.parameterize}-field")
```

Comment:
Could make this more "railsy" by doing `question.to_s.underscore` instead of the string substitution I think but this is fine too

Reply:
Thanks for the tip 😄 ****

Reply:
Oh, we had to use dasherize 😅